pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -351,11 +351,15 @@ var tpccSupportedWarehouses = []struct {
 	{hardware: "gce-n4cpu16", v: version.MustParse(`v2.1.0-0`), warehouses: 1300},
 	{hardware: "gce-n4cpu16", v: version.MustParse(`v19.1.0-0`), warehouses: 1250},
 	{hardware: "aws-n4cpu16", v: version.MustParse(`v19.1.0-0`), warehouses: 2100},
+	// TODO(darrylwong): this number is copied from gce-n4cpu16. Adjust once we have a
```

Comment:
Hmm, I actually don't remember what I did here, it doesn't match any of the numbers lol. I just know that the numbers "pass". 

Considering the values haven't been touched since 2019, I'll replace it with a more generic "look at all the numbers again" TODO.